tests/PolyType.Tests.NativeAOT/Program.cs

```diff
@@ -0,0 +1,93 @@
+using PolyType.Tests.NativeAOT;
+using System.Diagnostics;
+
+namespace PolyType.Tests.NativeAOT;
+
+// Console application entry point for Native AOT smoke tests
+internal class Program
+{
+    private static int Main(string[] args)
```

Comment:
A main method is unecessary, this boilerplate should all be handled using TUnit.